### ISSUE TITLE
Accessibility: Form control with "combobox" role has no associated label

### ISSUE THREAD

**Issue by loverdrive** [ng-select/ng-select#2437]
### Describe the bug

The role="combobox" to the div that wrap input is not properly set.

### Steps to Reproduce

1. Use ng-select component in the app.
2. Run "IBM Equal Access Accessibility Checker" scan with WCAG 2.1 (A, AA) Accessibility Guidelines

This is html generated by ng-select component
![image](https://github.com/user-attachments/assets/6ca3abb9-152a-4708-936a-eece52c6c4b7)

This is the report of accessibility violation:
![image](https://github.com/user-attachments/assets/41594edf-1dcf-4104-9f4b-3f1ac01d4f03)

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA16

### Expected behaviour

To implement one of the following solution:

![image](https://github.com/user-attachments/assets/61dfaca8-fce2-4138-bf64-5fdfd8142ef8)

### What version of the library you see this issue?

12.0.7

### What version of Angular are you using?

Angular 17

**Comment by pavankjadda** [comment]
Closed as part #2428
